docs/src/threading.rst

```diff
@@ -131,8 +131,14 @@ Functions return 0 on success or an error code < 0 (unless the
 return type is void, of course).
 
 .. note::
-    Callers should be prepared to deal with spurious wakeups on :c:func:`uv_cond_wait` and
-    :c:func:`uv_cond_timedwait`.
+    1. Callers should be prepared to deal with spurious wakeups on :c:func:`uv_cond_wait`
+       and :c:func:`uv_cond_timedwait`.
+    2. On z/OS, the time to wait for :c:func:`uv_cond_timedwait` specified by the timeout
```

Comment:
I appreciate the level of detail for z/OS. But this now begs the question: what does the `timeout` parameter do on all of the other OSes? It seems like it would be more fruitful to define the general behavior first, and then indicate the way in which z/OS deviates from the general behavior.

Reply:
I changed it so that the general behavior is described before the z/OS behavior.